curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/write-concise-declarative-functions-with-es6.english.md

```diff
@@ -53,6 +53,15 @@ console.log(bicycle.gear);
 
 </div>
 
+### After Test
+<div id='js-teardown'>
```

Comment:
```suggestion
<div id='js-setup'>
```
Same point again.

Reply:
It does not matter which section it goes in for this, so After Test and `tear-down` are fine.

Reply:
The title "After Test" has always bother med, because it is not really accurate.  It is not like the code in these blocks runs after all tests are complete.  `Before Test` only applies when you need to pass something to the tests before any of them run.

Reply:
It really suggests to me that it's code run after each test, like in a test suite.  Particularly when the same language as in unit tests - (setup/teardown).  I know that it *isn't*, but it looks that way.

Anyway, you've convinced me that it's a documentation issue, rather than a bug waiting to happen.  As such, I'm perfectly happy with this PR being merged.  I will see if there's an issue talking about this, as we can't be the only ones who've noticed this.